src/commands/kol/player.ts

```diff
@@ -0,0 +1,84 @@
+import {
+    ChatInputCommandInteraction,
+    SlashCommandBuilder,
+    bold, 
+    hyperlink
+  } from "discord.js";
+  
+import { kolClient } from "../../clients/kol";
+import { pluralize, toKoldbLink, toSnapshotLink } from "../../utils";
+
+
+export const data = new SlashCommandBuilder()
+  .setName("player")
+  .setDescription("Look up information on a given player.")
+  .addStringOption((o) =>
+    o
+      .setName("player")
+      .setDescription("The KOL username of the player you're looking up.")
+      .setRequired(true)
+    );
```

Comment:
consider allowing player ID as well. if that already works, consider adding it to the description